src/views/assets/Asset/AssetList/components/BaseList.vue

```diff
@@ -225,6 +226,11 @@ export default {
         row: row,
         payload: row.payload
       }
+
+      if (action === 'clone') {
+        return this.$refs.ListTable.onClone({ row, query: meta })
+      }
+
       this.$refs.ListTable.onUpdate({ row, query: meta })
     },
     createAsset(platform) {
```

Comment:
I'm sorry, I cannot see the exact code that needs to be checked for you. Please send over the code snippet.